lib/DDG/Spice/Symbolab.pm

```diff
@@ -7,9 +7,9 @@ spice is_cached => 1;
 spice to => 'https://www.symbolab.com/ddg?query=$1';
 spice wrap_jsonp_callback => 1;
 
-triggers startend => "calculate", "solve", "compute";
-handle remainder => sub {
-    if ($_ =~ /[\^\*\=\+\-\/\\]/ or $_ =~ /\b(sin|cos|tan|cot|csc|sec|ln|log|sqrt|integral)/i) {
+triggers startend => "calculate", "solve", "compute", "integral", "integrate", "integration", "antiderivative", "derivative", "derive", "differentiate", "derivation";
+handle query_lc => sub {
+    if ($_ =~ /[\^\*\=\+\-\/\\]/ or $_ =~ /\b(sin|cos|tan|cot|csc|sec|ln|log|sqrt|integr|deriv|diff)/i) {
```

Comment:
@levaly actually, you probably just need to add `\b` to the end of the 2nd regex unless you expect a specific character to follow those keywords?